scripts/apps/authoring-react/subcomponents/content-profile-dropdown.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import {IArticle, IContentProfile} from 'superdesk-api';
+import {gettext} from 'core/utils';
+import {Spacer} from 'superdesk-ui-framework/react';
+import {sdApi} from 'api';
+
+interface IProps<T> {
+    article: T;
```

Comment:
if it's article - it's not `T`, but `IArticle` and if it's `T` it's not an article :D `item` is probably the best name here.

Reply:
It's T, because if it's not the usage doesn't compile from authoring-react.

Reply:
yeah, I thought it's `T`, but then it means it's not an article - thus we should name the variable `item` and not `article`